documentation/SupportingDocuments/API-Purpose-Inventory.md

```diff
@@ -0,0 +1,5 @@
+# API-Purpose-Inventory
```

Comment:
This file is already included in PR https://github.com/camaraproject/IdentityAndConsentManagement/pull/43, isn't it?

Reply:
I removed it.. likely a commit error. Would be great if @jpengar can merge this PR as its pure lift and shift of authN-authZ doc